### ISSUE TITLE
Term Request: Vogt striae

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11773]
### Term Request: Vogt striae

**Preferred term label:**
Vogt striae

**Synonyms:**
* Vogt's striae
* Vogt lines
* Corneal stress lines
* Posterior stromal stress lines in keratoconus

**Definition (free text):**
Fine, parallel, vertically or obliquely oriented stress lines located in the deep posterior stroma immediately anterior to Descemet membrane, characteristically observed at the apex of the cone in keratoconus. They represent biomechanical tension lines induced by stromal thinning and steepening, and typically disappear transiently when gentle mechanical pressure is applied to the globe.

**Parent term (use hpo.jax.org/app):**
Abnormality of the corneal stroma (HP:0011492)

**Your nano-attribution (ORCID):**
0000-0001-9042-0508

**Comment by daboe01** [comment]
sorry for the dupe: HP:6001180
